livox_ros2_driver/livox_ros2_driver/lddc.h

```diff
@@ -95,7 +95,7 @@ class Lddc {
   std::shared_ptr<rclcpp::PublisherBase> GetCurrentImuPublisher(uint8_t handle);
   void PollingLidarPointCloudData(uint8_t handle, LidarDevice *lidar);
   void PollingLidarImuData(uint8_t handle, LidarDevice *lidar);
-  void InitPointcloud2MsgHeader(sensor_msgs::msg::PointCloud2& cloud);
+  void InitPointcloud2MsgHeader(sensor_msgs::msg::PointCloud2& cloud, const std::string& frame_id);
```

Comment:
I think if you don't write as `const std::string & frame_id` you get errors around `ament_cpplint` in `colcon test`, is that okay?

Reply:
How can I test?
I did `colcon build --symlink-install` and `colcon test` and share results.
https://gist.github.com/kosuke55/40312d603cb16ea5c30e76897b96848d

Reply:
How about using `colcon test --event-handlers console_cohesion+ --packages-select livox_ros2_driver`?

Reply:
livox_ros2_drive may has a lot of other lint error. so I think it is better to fix code style in other PR.